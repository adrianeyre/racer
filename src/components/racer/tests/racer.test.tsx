import React from 'react';
import { shallow } from 'enzyme';

import Racer from '../racer';
import IRacerProps from '../interfaces/racer-props';

describe('Racer Run', () => {
	it('Should render correctly', () => {
		const defaultProps: IRacerProps = {};
		const racer = shallow(<Racer {...defaultProps} />);
		expect(racer).toMatchSnapshot();
	});
});