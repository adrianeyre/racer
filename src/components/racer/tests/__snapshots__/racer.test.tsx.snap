// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Racer Run Should render correctly 1`] = `ShallowWrapper {}`;
